pkg/client_test.go

```diff
@@ -31,31 +31,35 @@ var TestClient = SecretClient{
 		}}}
 
 func TestSecretClient_GetSecret(t *testing.T) {
```

Comment:
Can we update these test functions to match the functions on the `ServiceClient`, basically pluralise. This also applies to the other functions in this test file.